src/components/Checklist.js

```diff
@@ -0,0 +1,42 @@
+import '../styles/global.css'
+import { useState } from "react";
+
+function Checklist() {
+    // Exercise 4: 
+    // use the useState hook to replace the checklist array, hint, each item will require key
```

Comment:
love that we are introducing the key idea for lists. however, I feel that it may be too complex at the moment? would it be okay to not have every list element have a key and have the state variable be a list rather than a dictionary?

Reply:
okay ill make the change! the only issue is that the console will error if we don't include a valid key. However, we can totally explain that away.